.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Description
+
+Please provide a brief summary of your changes along with a link to the related issue if applicable.
+
+#### Type of Change
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Improvement
+- [ ] Documentation update
+- [ ] Other (please describe)
+
+#### How Has This Been Tested?
+Provide information about the testing procedures, types of tests, and tools used to ensure the functionality and correctness of the changes.
+
+#### Checklist
+- [ ] I have read the [Contributing Guidelines](CONTRIBUTING.md).
+- [ ] I have followed the coding style and conventions of the project.
+- [ ] I have tested my changes and ensured that they work as expected.
+- [ ] I have updated any relevant documentation.
+- [ ] I have added tests that prove my fix or feature works.
+
+### Instructions for Creating a Pull Request
+
+1. **Fork the Repository**: Fork the Kaoto-next project to your personal GitHub account.
+2. **Clone the Repository**: Clone your fork to your local machine.
+3. **Create a Branch**: Make a new branch for your changes.
+4. **Make Your Changes**: Make and test your changes locally.
+5. **Commit and Push**: Commit and push your changes to your fork on GitHub.
+6. **Create a Pull Request**: Follow this template to create a pull request to the main Kaoto-next repository.
```

Comment:
@eerkmen, first of all, thanks for the contribution.

I think that at the time when this template is launched on the Pull Request page, the pull request is already created, so it seems that this text could be more appropriate for the `CONTRIBUTING.md` file.

What do you think?

```suggestion
```